cmd/kubeadm/app/cmd/phases/join/controlplanejoin.go

```diff
@@ -38,16 +38,31 @@ var controlPlaneJoinExample = cmdutil.Examples(`
 `)
 
 func getControlPlaneJoinPhaseFlags(name string) []string {
-	flags := []string{
-		options.CfgPath,
-		options.ControlPlane,
-		options.NodeName,
-	}
-	if name == "etcd" {
```

Comment:
isn't it simpler to just have this here?

```
if name == "etcd" || name == "all" {
```

Reply:
Yes, it is simpler! 
I originally referred to the format in `controlplaneprepare.go#getControlPlanePreparePhaseFlags` 😅